azure-pipelines.yml

```diff
@@ -0,0 +1,77 @@
+name: das-campaign-experiment
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+workspace:
+    clean: all
+
+variables:
+  buildConfiguration: 'release'
+  buildPlatform: 'anycpu'
+
+pool:
+  name: 'Continuous Integration 02 - SSD - 160ACU'
+  demands: 
+  - msbuild
+  - visualstudio
+
+steps:
+- task: gittools.gitversion.gitversion-task.GitVersion@4
```

Comment:
Version 5 is out now so can use that one 👍

Reply:
Updated to version 5
Had already moved file before you commented :-)